lib/util/eyeglass_modules.js

```diff
@@ -0,0 +1,283 @@
+"use strict";
+
+var resolve = require("./resolve");
+var packageUtils = require("./package");
+var path = require("path");
+var merge = require("lodash.merge");
+var includes = require("lodash.includes");
+var semver = require("semver");
+var uuid = require("uuid");
+
+var EYEGLASS_KEYWORD = "eyeglass-module";
+var ROOT_ID = ":root-" + uuid.v4();
```

Comment:
I was going somewhere with that, but I backtracked and probably don't need it anymore.